lib/_autoload_modules.php

```diff
@@ -24,7 +24,7 @@ function SimpleSAML_autoload($className)
     $module = substr($className, $modulePrefixLength, $modNameEnd - $modulePrefixLength);
     $moduleClass = substr($className, $modNameEnd + 1);
 
-    if (!SimpleSAML_Module::isModuleEnabled($module)) {
+    if (getenv('SIMPLESAMLPHP_ENABLE_ALL_MODULES') == false && !SimpleSAML_Module::isModuleEnabled($module)) {
```

Comment:
This is a bit confusing. This change has nothing to do directly with the PR itself, and honestly I don't think you need it or even that it is a good idea.

Also, strict comparison operator should be used in a case like this.